lib/AST/Builtins.cpp

```diff
@@ -1018,6 +1018,12 @@ static ValueDecl *getDeallocOperation(ASTContext &ctx, Identifier id) {
                             _void);
 }
 
+static ValueDecl *getStackAllocOperation(ASTContext &ctx, Identifier id) {
+  return getBuiltinFunction(ctx, id, _thin,
+                            _parameters(_word, _word),
+                            _rawPointer); // FIXME: return _optional(_rawPointer)
+}
```

Comment:
* [`createAsyncTaskGroupFuture`](https://github.com/apple/swift/blob/a53ab388bbbe86db37fd388201781f6a50f414c4/include/swift/AST/Builtins.def#L817) has a `Builtin.RawPointer?` parameter.

* [`getCreateAsyncTaskGroupFuture`](https://github.com/apple/swift/blob/a53ab388bbbe86db37fd388201781f6a50f414c4/lib/AST/Builtins.cpp#L1442-L1443) uses the `BuiltinFunctionBuilder` class.

```suggestion
static ValueDecl *getStackAllocOperation(ASTContext &ctx, Identifier id) {
  BuiltinFunctionBuilder builder(ctx);
  builder.addParameter(makeConcrete(ctx.getIntType())); // byteCount
  builder.addParameter(makeConcrete(ctx.getIntType())); // alignment
  builder.setResult(makeConcrete(OptionalType::get(ctx.TheRawPointerType)));
  return builder.build(id);
}
```

Reply:
Thank you for the suggestion, Ben. I'm aware of those symbols. You'll note that `createAsyncTaskGroupFuture()` generates an optional value in the standard library, not in the builtin itself.